docs/open-api.yaml

```diff
@@ -28,7 +28,7 @@ paths:
       x-amazon-apigateway-integration:
         uri:
           Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${FetchJournalByIdentifierAndYearFunction.Arn}/invocations
-        httpMethod: POST
+        httpMethod: GET
```

Comment:
Jeg tror at her i integrasjonen mellom API Gateway og lambda skal det alltid være POST. Den reelle HTTP-metoden for selve kallet vil gjenspeiles her i selve open-api-spekken og i template.yaml.

Reply:
Fixed